r/restrserve/.Rprofile

```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```

Comment:
I would remove `renv` related stuff as it is pointless to use it with our basic setup.

Reply:
ok